sips/sip-031/sip-031.md

```diff
@@ -0,0 +1,575 @@
+> - **Sui**: Since the announcement of its ecosystem fund worth over $50M to support its DeFi ecosystem in Q4 2023, **Sui** has seen explosive growth, with TVL increasing by \~50x, DEX volumes by \~20x, and daily active users \~15x as of Q1 2025\. The comparison shows clear growth across all key metrics, which could suggest that the initial funding helped drive ecosystem growth.
+> - **Avalanche**: Since the announcement of its $180M DeFi incentive program in Q3 2021, **Avalanche** has grown steadily, with TVL increasing by \~8x, DEX volumes by \~2x, and daily active users by \~6x as of Q1 2025\. The comparison shows clear growth across all key metrics, which could suggest that the initial funding helped drive ecosystem growth.
+> - **NEAR**: Since the announcement of its $800M funding initiative in Q4 2021, **NEAR** has seen significant growth, with TVL increasing by \~4x, fees by \~3x, and daily active users by \~160x as of Q1 2025\. The comparison shows clear growth across all key metrics, which could suggest that the initial funding helped drive ecosystem growth.
+
+
+### **Treasury Impact**
+
+- **Increased development activity:** Consistent funding for core protocol developers to drive development and innovation.  
+- **Ecosystem growth:** Grants and incentives will attract developers, projects, and users to the Stacks ecosystem.  
+- **Enhanced marketing and adoption:** Dedicated resources towards marketing will increase awareness and adoption of Stacks.  
+- **Improved market dynamics:** Active treasury management and more robust capital markets activities can contribute to a healthier STX market.
+
+### **Treasury Allocations & Usage**
+
+**Building for the Long Run: Roadmap and Stacks Value Accrual**
+
+To more deeply understand the technical and growth efforts this SIP intends to support, we invite voters and stakeholders to read the following key resources:
+
+* [Stacks Roadmap Update](https://stacksroadmap.com)  
+* [Stacks Economic Model: Unlocking BTC Capital & Long Term Growth](https://forum.stacks.org/t/stacks-economic-model-unlocking-bitcoin-capital-long-term-growth/18035)  
+* [sBTC Design Principles](https://bitcoinl2labs.com/sbtc-design-principles)  
+* [Stacks DeFi Growth Roadmap](https://forum.stacks.org/t/2025-roadmap-defi-road-to-1bn-defi-tvl/18032)
+
+If approved, this SIP calls for the treasury funds to be allocated in accordance with the roadmap and other resources linked directly above. Taking a look at the top-line budget, it will be deployed across several key areas with the following approximate breakdown:
+
+### **Stacks Endowment Initial Budget Allocation**
```

Comment:
Should this be a sub-heading of "Treasury Allocations & Usage"?  Because, it's not right now.